tools/ci/checks/run_black_checks.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+DOCKER_TAG=${DOCKER_TAG:-latest}
+DOCKER_IMAGE=${DOCKER_IMAGE:-mesosphere/dcos-commons:${DOCKER_TAG}}
+
+if [ x"$1" == x"--format" ]; then
```

Comment:
Please drop the x, or drop the quotes and use ``[[ ]]` better yet.

Reply:
I will drop the file.